# A1_DatesetIris
Luis Humberto Sánchez Vaca
A01638029

# Importing and inspecting the data

In [1]:
# Import the packages that we will be using
import matplotlib.pyplot as plt
import pandas as pd                 # For data handling

# Dataset url
url = "datasets/iris/iris.csv"

# Load the dataset
dataset = pd.read_csv(url, names=['SepalLength','SepalWidth','PetalLength','PetalWidth','Class'])

In [2]:
dataset

SepalLength  SepalWidth  PetalLength  PetalWidth           Class
0            5.1         3.5          1.4         0.2     Iris-setosa
1            4.9         3.0          1.4         0.2     Iris-setosa
2            4.7         3.2          1.3         0.2     Iris-setosa
3            4.6         3.1          1.5         0.2     Iris-setosa
4            5.0         3.6          1.4         0.2     Iris-setosa
..           ...         ...          ...         ...             ...
145          6.7         3.0          5.2         2.3  Iris-virginica
146          6.3         2.5          5.0         1.9  Iris-virginica
147          6.5         3.0          5.2         2.0  Iris-virginica
148          6.2         3.4          5.4         2.3  Iris-virginica
149          5.9         3.0          5.1         1.8  Iris-virginica

[150 rows x 5 columns]

# Activity: work with the iris dataset

1. Load the iris.csv file in your computer and compare with the data printed above.
    
    The data printed here is presented with a better format like a table, but it doesn't display all the data at the same time, just a few rows at the begining and some other at the end.


2. How many observations (rows) are in total?

    There are 150 observations.


3. How many variables (columns) are in total? What do they represent?

    There are 5 variables: Sepal length, Sepal width, Petal lenght, Petal width and Class.


4. How many observations are for each type of flower? 

    There are 50 observations for each type of flower.


5. What is the type of data for each variable?

    The first four variables are float numbers, and the last one is a string.


6. What are the units of each variable?

    The first four variables are in cm, and the last one doesn't have units.